include/evmc/helpers.h

```diff
@@ -17,6 +17,14 @@
 
 #include <evmc/evmc.h>
 
+/**
+ * Returns true if the instance has a compatible ABI version.
+ */
+static inline int evmc_has_compatible_abi(struct evmc_instance *instance)
```

Comment:
Not sure about how to nicely name this function.

Reply:
`evmc_is_abi_compatible()`?

Reply:
Changed.